.travis.yml

```diff
@@ -1,6 +1,7 @@
 language: go
+go_import_path: github.com/projectriff/riff
```

Comment:
this is the default value, do we need to override it?

Reply:
that's the only way I could build `fbiville/riff` in Travis CI.
Without `go_import_path`: https://travis-ci.org/fbiville/riff/builds/425791314
With it: https://travis-ci.org/fbiville/riff/builds/425795890

Reply:
> forks won't have the correct env var to allow the build to pass anyway. They should provide their own CI
What do you mean @scothis ?

In any case, forks would compile in a package that is indeed `github.com/projectriff/riff`, so I think this setting is sound

Reply:
Forks that are expected to merge back upstream should use the main travis config and build via PRs. Permanent forks should find their own CI tools.

The `.travis.yaml`file is not expected to work when run from any repo other than `projectriff/riff`. There are a number of environment variables that are setup within our config that a fork will not have and should never have.

I don't want to support CI for forks of riff. PR builds work great, use them. It's fine to open a PR and tag it as DNM or Do Not Merge for in progress work.

Reply:
Ok, so as you say, to even test a [future] PR against fats, one then needs to _actually_ open a PR and make changes, push, observe, rinse, repeat. That's fine by me.

Let's drop this line then